app/scripts/metamask-controller.js

```diff
@@ -116,16 +116,34 @@ export default class MetamaskController extends EventEmitter {
     })
 
     // This depends on preferences controller state
-    this.trackSegmentEvent = getTrackSegmentEvent(
+    this.trackSegmentEvent = getTrackMetaMetricsEvent(
```

Comment:
Oh, one more: I think we can remove the comment, which is useless, and then we should probably rename `MetaMaskController.trackSegmentEvent` to `trackMetaMetricsEvent`.